### ISSUE TITLE
Building dist bundle for production without building Electron app

### ISSUE THREAD

**Issue by alectrocute** [nklayman/vue-cli-plugin-electron-builder#748]
**Describe the bug**

When I run `npm run electron:build --legacy --mac`, the plugin creates a wonderfully operational `/dist_electron/bundled` folder, which works with CORS and remote execution from Electron.

However, when I run `npm run build` or `vue-cli-service build`, the bundled files contain errors related to `fs`.

I'd really love to see a `npm run electron:build --bundle-only` command. And in the meantime, if possible, some workaround.

Thank you so much for this life-saving project.

**Comment by alectrocute** [comment]
@nklayman Any idea how to do a workaround for this for an urgent deadline?

**Comment by nklayman** [comment]
Are you trying to generate a build that is used in a browser? I'm not quite sure what you are trying to achieve. If you are getting errors about fs in the `npm run build` it is likely because you are importing fs itself or a module that relies on it. If this is the case, you will have to stub out that module for browser builds like in https://github.com/nklayman/vue-cli-plugin-electron-builder/issues/723#issuecomment-616722646.

**Comment by alectrocute** [comment]
I was worried I did an awful job explaining what I think is actually a feature request... Let me try again, sorry.

> Are you trying to generate a build that is used in a browser? 

Yes and no. Objectively - I'm trying to bundle the app exactly how it would be if it were headed for electron-builder, but then exit prematurely before the handoff. Subjectively - I'm attempting to create a single build that will run inside Electron both internally and externally (eg. `window.loadURL("file://index.html#/mycoolpage")` vs. `window.loadURL("https://my-secure-site-vpn-etc-etc.com/index.html#/mycoolpage")`

**Comment by alectrocute** [comment]
Related: #212

**Comment by alectrocute** [comment]
Currently, I get around this by just building for Linux and then copying `./dist_electron/bundled` into a .zip. Works great. It just takes time building a Linux build, it would be massively convenient to have a `--bundle-only` flag to build with v-cli-p-e-b's config but not actually pass anything off to electron-builder.

**Comment by nklayman** [comment]
If you use `npm run electron:build -- --linux dir` it should only take a few seconds extra to build since it only needs to create the `linux-unpacked` build. I will add a `--onlyBundle` flag but for now you can just use the workaround.

**Comment by alectrocute** [comment]
Just what I needed, thanks much @nklayman!

**Comment by nklayman** [comment]
I don't think I'll add a flag as the use case is pretty rare and all it will save is a second or two. If you are happy with the current solution then I will leave it as is.
